Tests/Unit/EventListener/SessionListenerTest.php

```diff
@@ -47,6 +47,28 @@ public function testOnKernelRequestRemainsUntouched()
         $listener->onKernelRequest($event);
     }
 
+    public function testOnFinishRequestRemainsUntouched()
```

Comment:
The test should be skipped if the method does not exist in the core session listener. See https://travis-ci.org/FriendsOfSymfony/FOSHttpCacheBundle/jobs/396800434

Reply:
@Toflar Done!